code/modules/lavaland/lavaland_theme.dm

```diff
@@ -12,6 +12,20 @@
 	else if(!ispath(primary_turf_type))
 		stack_trace("Wrong turf type `[primary_turf_type.type]` in `[type]` lavaland theme")
 
+/datum/lavaland_theme/proc/setup_caves()
+	var/max_attempts = 100
+	var/max_cave_spawns = 40
+	var/z = level_name_to_num(MINING)
+	while(max_attempts > 0 && max_cave_spawns > 0)
+		var/x = rand(1, 255)
+		var/y = rand(10, 222)
```

Comment:
These feel somewhat like magic numbers, do we have these in a define somewhere?

Reply:
I've updated these to use world.maxx and defines where appropriate, PTAL (also why isn't there a world.minx)